internal/jobs/launch_instance_azure.go

```diff
@@ -154,6 +158,10 @@ func DoLaunchInstanceAzure(ctx context.Context, args *LaunchInstanceAzureTaskArg
 		Pubkey:            pubkey,
 		InstanceType:      clients.InstanceTypeName(reservation.Detail.InstanceSize),
 		UserData:          userData,
+		Tags: map[string]*string{
+			"rh-rid": ptr.To(config.EnvironmentPrefix("r", strconv.FormatInt(reservation.ID, 10))),
```

Comment:
Just a reminder that this is a pointer but we do not return nil anywhere, so it is okay for now.

Reply:
Yeah it is because Azure requires exactly this type, otherwise I would need to copy the values manually.